jbehave-support-core/src/main/resources/report-generator/serverLogXmlRep.xslt

```diff
@@ -41,63 +46,108 @@
                 <xsl:number level="any"/>
             </xsl:variable>
 
-            <div>
-                <table width="100%" class="sshLog">
-                    <tbody>
-                        <tr>
-                            <td width="15%" class="label">Start log:</td>
-                            <td width="85%" class="time-string-millis">
-                                <xsl:value-of select="@startDate"/>
-                            </td>
-                        </tr>
-                        <tr>
-                            <td class="label">End log:</td>
-                            <td class="time-string-millis">
-                                <xsl:value-of select="@endDate"/>
-                            </td>
-                        </tr>
+            <table width="100%" class="sshLog">
+                <tbody>
+                    <tr>
+                        <td width="15%" class="label">Start log:</td>
+                        <td width="85%" class="time-string-millis">
```

Comment:
boostrap > try it for styles

Reply:
> Hi @mbocek I was investigating AbstractXmlReporterExtension.printString() method due to exception catching. It is called by every print method of report extensions and can throw IllegalStateException. But it seems that if this exception is thrown anywhere report is going to be broken. So there is no point in trieing to catch it in ReportExtension.

In case you are right, we should create new issue for this particular behavior.